optimum/intel/ipex/modeling_base.py

```diff
@@ -304,7 +349,7 @@ def forward(
         if "token_type_ids" in self.input_names:
             inputs["token_type_ids"] = token_type_ids
 
-        outputs = self._call_model(**inputs)
+        outputs = self.model(**inputs)
```

Comment:
would prefer to keep this changes for https://github.com/huggingface/optimum-intel/pull/566 as we already have the discussion there